modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner (HPP) custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, you are not required to change your HPP custom resource. You continue using your existing storage classes. You do not have to modify the HPP custom resource for the legacy provisioner DaemonSet.
```

Comment:
Again the daemonset part is not relevant, that is an implementation detail, not a configuration detail.

Reply:
So if you upgrade an existing hpp to 4.10. It will update the 'legacy' provisioner AND create the csi driver based on the contents of the 'legacy' hpp CR. In order to use the CSI driver, you have to create the appropriate storage class. That is all you need to do when upgrading. For a new installation you should create a new HPP CR in the new format.

Reply:
How about this for the intro section? I want to differentiate the user action for upgrading and installing on a fresh cluster. Adding Alexander's comments and parsing out the deamonset verbiage, I'm wondering if we shouldn't leave the discussion of storage classes and unique names to the procedure section.

_If you upgrade to {VirtProductName} version 4.10 on an existing cluster, the system will update the legacy host path provisioner (HPP) custom resource as well as create a CSI driver custom resource based on the contents of the legacy HPP custom resource. 

If you are installing {VirtProductName} version 4.10 on a new cluster, you must install a CSI custom resource that uses the Kubernetes CSI driver._